docs/UserGuide.md

```diff
@@ -171,7 +171,7 @@ Parameters:
 Example usages:
 * `add n/Shingeki no Kyojin`
 * `add n/Shingeki no Kyojin e/2`
-* `add n/Shingeki no Kyojin s/w`
+* `add n/Shingeki no Kyojin s/w` (Tip: `w` here is short for the status `watching`. See the [`STATUS`](#status) section for more details.)
```

Comment:
this should have been explained above, as adding a tip here breaks the structure of example usages

Reply:
at this point we have not yet introduced status. i am assuming our tutor is gonna read our ug in order so i added that, but i get ur point